cmd/client/wallet/main.go

```diff
@@ -324,6 +326,26 @@ func processExportCommand() {
 	}
 }
 
+func processBlsgenCommand() {
+	newCommand.Parse(os.Args[2:])
+	noPass := *newCommandNoPassPtr
+	// Default password is an empty string
+	password := ""
+
+	if !noPass {
+		password = utils.AskForPassphrase("Passphrase: ")
+		password2 := utils.AskForPassphrase("Passphrase again: ")
+		if password != password2 {
+			fmt.Printf("Passphrase doesn't match. Please try again!\n")
+			os.Exit(3)
+		}
+	}
+
+	privateKey, fileName := blsgen.GenBlsKeyWithPassPhrase(password)
+	fmt.Printf("Bls private key: %s\n", privateKey.GetHexString())
```

Comment:
Please print out the public key here.  That's the key user needs to send to us so that we can add it into our genesis block. @rlan35 should know it.